.github/actions/deploy/action.yaml

```diff
@@ -52,7 +52,7 @@ runs:
       shell: bash
       run: bash deploy-zenko.sh end2end default
       working-directory: ./.github/scripts/end2end
-    - name: Add Keycloak user
+    - name: Add Keycloak user and assign StorageManager role
```

Comment:
Picking random line:

It would be great to create a markdown page that explains how to contribute to the CTST tests in Zenko CI, so that we can also explain how they are supposed to work, i.e.:

- The tests are idempotent: they must not conflict with each other, here by using random bucket/object names.
- It should be possible to rerun the tests after the end of the CTST test suite (so, tests are not dependent on the environment).
And then maybe some other rules to follow, what do you think?

Reply:
a1872b532f5fd6ea0924a55df3623d859e43baba

thoughts?

Reply:
> Beyond usual Cucumber.js practice of using worlds, steps and features, this test suite has some specifics to consider:

This is something all tests should have in common, this is not specific to CTST, but an important reminder yes!

You could also add that CTST comes along with the AWS-CLI tool and abstraction layers to run it from TS code, but can be extended if needed (iwht iether cli tools or pure http tools, like axios).